docs/source/whatsnew/pr/autoreload-verbosity.rst

```diff
@@ -0,0 +1,21 @@
+Autoreload verbosity
+====================
+
+We introduce more descriptive names for the `%autoreload` parameter:
```

Comment:
Two suggestions:
- replace all single backticks with double backtics
- add a new line here so that the list renders nicely

rst has quite specific requirements.

```suggestion
We introduce more descriptive names for the ``%autoreload`` parameter:

```

Reply:
Sorry about that. I rendered the RST and think I got it right now.